src/lighting/TrainLighting.cpp

```diff
@@ -0,0 +1,307 @@
+#include "TrainLighting.hpp"
+
+namespace godot {
+    const char *TrainLighting::WRAPPER_POSITION_CHANGED_SIGNAL = "wrapper_position_changed";
```

Comment:
why this is called "wrapper_position_changed"? why not "selector_position_changed"?